contrib/quickstart/kratos/email-password/kratos.yml

```diff
@@ -94,4 +94,4 @@ identity:
 
 courier:
   smtp:
-    connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true
+    connection_uri: smtps://test:test@mailslurper:1025/?skip_ssl_verify=true&legacy_ssl=true
```

Comment:
Question: Is it needed? It looks like quick-start configuration.
With our vanilla setup it should not be needed anymore

Reply:
Yeah, it's not needed. Removed.